frontend/src/pages/Layout.tsx

```diff
@@ -1,25 +1,21 @@
 import React from 'react';
-import { BrowserRouter, Routes, Route } from "react-router-dom";
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
 
 import PATHS from '../constants/routes';
-import Header from "../components/layout/Header"
-import Menu from "../components/layout/Menu"
+import Header from '../components/layout/Header';
+import Menu from '../components/layout/Menu';
```

Comment:
Do we have already absolute paths?
Configure please prettier or lint to avoid fixing the quotes.